### PR TITLE
Updated Android Studio status, added native preview to IntelliJ IDEA

### PR DESCRIPTION
**Pull request by tonyunreal**
Updated Android Studio status, added native preview to IntelliJ IDEA

### CODE REVIEW

**Comment by ThatGuySam** [comment]
I like this note so people know about the previews. 

For Android Studio, let's add the word _only_ so we're still communicating that the emulator in the stable release still doesn't work: 

✳️ Yes, works via Rosetta 2, emulator only available as preview

**Comment by richiksc** [comment]
Fixes #422.

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Any comments, suggestions? [Let us know!](https://github.com/ThatGuySam/doesitar
 * [Haskell](https://www.haskell.org/platform/mac.html) - 🚫 Not yet supported only works on Intel-based Macs - [Gitlab Issue](https://gitlab.haskell.org/ghc/ghc/-/issues/18664)
 * [Homebrew](https://brew.sh/) - ✳️ Yes, with caveats and some troubleshooting. - [Issue](https://github.com/Homebrew/brew/issues/7857). 
 * [Hopper Disassembler](https://www.hopperapp.com/download.html) - ✅ Yes, Full Native Apple Silicon Support as of v4.6 - [Release Notes](https://www.hopperapp.com/blog/?p=263)
-* [IntelliJ Idea](https://www.jetbrains.com/idea/download/#section=mac) - ✳️ Runs via Rosetta with native support currently in development - [Official Jetbrains Issue](https://youtrack.jetbrains.com/issue/JBR-2526)
+* [IntelliJ Idea](https://www.jetbrains.com/idea/download/#section=mac) - ✳️ Runs via Rosetta 2, native support available as preview - [Official Jetbrains Issue](https://youtrack.jetbrains.com/issue/JBR-2526) [Download Preview](https://youtrack.jetbrains.com/issue/JBR-2526#focus=Comments-27-4589077.0-0)
```
Comment (richiksc) [inline README.md:50]: Maybe change this to "IntelliJ IDEA" while you're at it?

Comment (ThatGuySam) [reply README.md]: Done

**Review by richiksc — APPROVED** [review]

**Pull request merged**